tests/plugins/tool/black_tool_plugin/valid_package/format_errors.py

```diff
@@ -1 +1 @@
-x =  "some string",
+x = "some string",
```

Comment:
Looks like we want to revert/undo this change.

Reply:
I wish this would fail a unit test. I'll look into that later.

Reply:
a132952